### ISSUE TITLE
Sort order not properly working for DEs

### ISSUE THREAD

**Issue by ifoche** [EyeSeeTea/dhis2-android-eventcapture#63]
The report from the alpha testers is as follows:

I believe that the Web client correctly takes a program's data element order from "Section Management", while Android takes DE order only from "Program Stage Management", ignoring Section Management.
DE organisation in "Section Management" should override the order set out in "Program Stage Management", so this bug is in Android and the Web client is functioning properly.
To summarize:
Data Elements on the Web are arranged in accordance with the order set out in "Section Management"
Data Elements on Android are organised in the order set out in "Program Stage Management"
